packages/contracts-bedrock/src/libraries/Types.sol

```diff
@@ -75,4 +75,22 @@ library Types {
         L1,
         L2
     }
+
+    /// @notice Enum representing different types of configurations that can be set on L1BlockInterop.
+    /// @custom:value GAS_PAYING_TOKEN  Represents the config type for setting the gas paying token.
+    /// @custom:value BASE_FEE_VAULT_CONFIG Represents the config type for setting the base fee vault configuration.
+    /// @custom:value L1_FEE_VAULT_CONFIG  Represents the config type for setting the L1 fee vault configuration.
+    /// @custom:value SEQUENCER_FEE_VAULT_CONFIG Represents the config type for setting the sequencer fee vault
+    ///               configuration.
+    /// @custom:value ADD_DEPENDENCY        Represents the config type for adding a chain to the interop dependency set.
+    /// @custom:value REMOVE_DEPENDENCY     Represents the config type for removing a chain from the interop dependency
+    /// set.
+    enum ConfigType {
+        GAS_PAYING_TOKEN,
```

Comment:
two important things to point out here:
- Are we sure whats the impact on renaming this from `SET_GAS_PAYING_TOKEN` to `GAS_PAYING_TOKEN`
- Namings dont look consistent between them right now since you have for example `GAS_PAYING_TOKEN` and `ADD_DEPENDENCY`. On the first one you dont have a verb/action on the name, but on the second one you do have "add" in it. What do you think about keeping the convetion of having the verb/action in it.